__init__.py

```diff
@@ -430,7 +430,7 @@ def handle_is_it_foggy(self, message: Message):
         self._report_weather_condition(message, "fog")
 
     @intent_handler(
-        IntentBuilder("").require("ConfirmQuery").require("Rain").optionally("location")
+        IntentBuilder("").require("ConfirmQuery").require("Rain").optionally("Location")
```

Comment:
Is capitalized

Reply:
Ah it looks like this got changed in a recent commit from @chrisveilleux that's not yet on your fork.